clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2774,7 +2784,9 @@ private String printJobStatus(
 
     Optional<String> details = response.getOptionalStatusDetails();
     if (details.isPresent() && detailsAreDifferent(previousOverallDetails, details.get())) {
-      LOGGER.info("\t\tNew overall details: {}", details.get());
+      if (!details.get().isEmpty()) {
```

Comment:
nit: could be cleaner if we move this check to `detailsAreDifferent`. Feel free to drop

Reply:
logic wont be same as in the next line it returns the `newOverallDetails`